lib/bec_import.rb

```diff
@@ -0,0 +1,57 @@
+class BecImport
+  def initialize(lines)
+    @lines = lines
+    validate!
+  end
+
+  def delete_unused
+    BusinessEntity.all.each do |be|
+      be.destroy if to_delete?(be)
```

Comment:
What is going to happen if an office has changed their jurisdictions since the spreadsheets were generated?